### PR TITLE
AUV - remove the use of the .dir_manifest option. using .manifest wit…

### PR DESCRIPTION
**Pull request by lbesnard**
…h 2**12 lines per manifest

### CODE REVIEW

**Comment by lbesnard** [comment]
linked to https://github.com/aodn/data-services/pull/1017 

This is my attempt at slowly reducing pain I'm inducing to our beloved @lwgordonimos

File: test_aodndata/auv/test_auv.py [hunk 1]
```diff
@@ -70,15 +84,17 @@ def test_netcdf_product_manifest_file(self):
...
-    def test_pdf_dir_manifest_file(self):
+    def test_pdf_manifest_file(self):
         manifest_file = os.path.join(self.temp_dir,
-                                     'WA201705-r20170526_080319_SS14_geebank_36m_out.pdfreports.dir_manifest')
+                                     'WA201705-r20170526_080319_SS14_geebank_36m_out.pdfreports.manifest')
         test_report_path = os.path.join(self.temp_dir, 'AUV/AUV_DOWNLOAD_CAMPAIGN/WA201705/all_reports')
         shutil.copytree(os.path.join(TEST_ROOT, 'WA201705'),
                         os.path.join(self.temp_dir, 'AUV/AUV_DOWNLOAD_CAMPAIGN/WA201705'))
 
+        file_list = list_recursively_files_abs_path(os.path.join(test_report_path))
```
Comment (lwgordonimos) [inline test_aodndata/auv/test_auv.py:94]:
```suggestion
        file_list = list_recursively_files_abs_path(test_report_path)
```

File: test_aodndata/auv/test_auv.py [hunk 2]
```diff
@@ -88,14 +104,16 @@ def test_pdf_dir_manifest_file(self):
...
 
     def test_dive_dir_manifest_file(self):
-        manifest_file = os.path.join(self.temp_dir, 'WA201705-r20170526_080319_SS14_geebank_36m_out.dive.dir_manifest')
+        manifest_file = os.path.join(self.temp_dir, 'WA201705-r20170526_080319_SS14_geebank_36m_out.1.dive.manifest')
         test_dive_path = os.path.join(self.temp_dir,
                                       'AUV/AUV_DOWNLOAD_CAMPAIGN/WA201705/r20170526_080319_SS14_geebank_36m_out')
         shutil.copytree(os.path.join(TEST_ROOT, 'WA201705'),
                         os.path.join(self.temp_dir, 'AUV/AUV_DOWNLOAD_CAMPAIGN/WA201705'))
 
+        file_list = list_recursively_files_abs_path(os.path.join(test_dive_path))
```
Comment (lwgordonimos) [inline test_aodndata/auv/test_auv.py:113]:
```suggestion
        file_list = list_recursively_files_abs_path(test_dive_path)
```

**Comment by lwgordonimos** [comment]
Doesn't cause me too much pain, but it will help avoid handlers taking 5 days to process some files, which I think is more painful for you ;)

File: test_aodndata/auv/test_auv.py (hunk 2 above)
Comment (lbesnard) [reply test_aodndata/auv/test_auv.py]: ah yeah, thanks for that!

**Pull request merged**